manifests/p/Python/Python/3/3.10.5/Python.Python.3.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Python.Python.3
+PackageVersion: 3.10.5

```

```suggestion
PackageVersion: 3.10.5150.0
```
This must match Control Panel.